lib/models/addon.js

```diff
@@ -175,6 +175,15 @@ function registryHasPreprocessor(registry, type) {
   @param {Project} project The current project (deprecated)
 */
 let addonProto = {
+
+  /**
+  Initializes the addon.  If you override this method make sure and call `this._super()` or your addon will not work.
+
+  @public
+  @method init
+  @param {Project|Addon} parent The project or addon that directly depends on this addon
+  @param {Project} project The current project (deprecated)
```

Comment:
could you add an `@example` for this method? I think the `this._super()` call should actually include the `arguments` too (e.g. like in https://github.com/babel/ember-cli-babel/blob/4ab3d4a0929c6a4c9420a7bd6c15ed24f6d976f6/index.js#L23)